src/pudl/transform/ferc1.py

```diff
@@ -3151,6 +3151,23 @@ def process_dbf(self, raw_dbf):
         raw_dbf = super().process_dbf(raw_dbf)
         return raw_dbf
 
+    def transform_main(self: Self, df: pd.DataFrame) -> pd.DataFrame:
+        """Drop duplicate records from f1_income_stmnt.
+
+        Because net_utility_operating_income is reported on both page 1 and 2 of the
+        form, it ends up introducing a bunch of duplicated records, so we need to drop
+        one of them. Since the value is used in the calculations that are part of the
+        second page, we'll drop it from the first page.
+        """
+        df = super().transform_main(df)
+        df = df[
+            ~(
+                (df.record_id.str.startswith("f1_income_stmnt_"))
+                & (df.income_type == "net_utility_operating_income")
+            )
+        ]
+        return df
```

Comment:
why were these duplicate records not getting caught up in our PK checks for the table? pk is:
```
"primary_key": [
                "utility_id_ferc1",
                "report_year",
                "utility_type",
                "income_type",
            ],
```